starter-code/index.html

```diff
@@ -3,40 +3,64 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
 </head>
 <body>
+<div class="wrapper">
+  <div class="header">
+    <div class="logo-div"></div>
+  </div>
+  <div class="mid-container"></div>
+  <div class="imgs">
+    <p class="titleSpot">What’s on Spotify?</p>
+    <div class="first-img">
```

Comment:
instead of setting one class for each elements, you can set just one class for all of three divs, then you can style them reusing code!